misc.h

```diff
@@ -37,19 +37,114 @@
 #endif
 
 #define IS_MR_CHANNEL(x)       ((x) <= MR_CHANNEL_LAST)
-#define IS_FREQ_CHANNEL(x)     ((x) >= FREQ_CHANNEL_FIRST && (x) <= FREQ_CHANNEL_LAST)
+#define IS_VFO_CHANNEL(x)     ((x) >= VFO_CHANNEL_FIRST && (x) <= VFO_CHANNEL_LAST)
 #define IS_VALID_CHANNEL(x)    ((x) < LAST_CHANNEL)
 #define IS_NOAA_CHANNEL(x)     ((x) >= NOAA_CHANNEL_FIRST && (x) <= NOAA_CHANNEL_LAST)
 
+#ifdef ENABLE_ALT_CHANNELS
+	typedef uint16_t channel_t;
+
+	#define EEPROM_SIZE              0xFFFF  //Exact size is 0x10000 but the ckecs are "gret or equal" so a bit off to fit in 2 bytes looks like ok
+	#define VFO_CHANNELS_COUNT       7       // VFO in each freq band is like a "system channel"
+	#define FM_CHANNELS_COUNT        20      // are these NOAA? no clue, but let's define them in a constant here
```

Comment:
These are not NOAA, these are FM broadcast radio channels. You don't need that for your modification.

Reply:
Entire NOAA is confusing me, but I think with hints from the chirp driver more pieces fit in place.